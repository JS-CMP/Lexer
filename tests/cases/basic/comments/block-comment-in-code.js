// Block comment within code
var f = /* comment */ "world";