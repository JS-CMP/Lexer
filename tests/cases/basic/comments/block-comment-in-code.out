#include "types/JsAny.hpp"
#include "global/global.hpp"
#include "types/objects/Types.hpp"
#include "customOperators/CustomOperators.hpp"

int main() {
    JS::Any f;
    f = JS::Any(u"world");
    return 0;
}
